aspnetcore/fundamentals/configuration/sample/src/ObjectGraph/ObjectGraph.csproj

```diff
@@ -1,18 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
-    <AssemblyName>ObjectGraph</AssemblyName>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
     <OutputType>Exe</OutputType>
```

Comment:
I believe this `OutputType` node can be removed.

Reply:
... for `Sdk.Web` projects; this is an `Sdk` project (console app).

Reply:
@guardrex Nice catch! I overlooked that.